chain/jsonrpc/src/lib.rs

```diff
@@ -718,6 +719,14 @@ impl JsonRpcHandler {
         let (block_id,) = parse_params::<(MaybeBlockId,)>(params)?;
         jsonify(self.view_client_addr.send(GetValidatorInfo { block_id }).await)
     }
+
+    /// Returns the current epoch validators ordered in the block producer order with repetition.
+    /// This endpoint is solely used for bridge currently and is not intended for other external use
+    /// cases.
+    async fn validators_ordered(&self, params: Option<Value>) -> Result<Value, RpcError> {
+        let (block_id,) = parse_params::<(MaybeBlockId,)>(params)?;
```

Comment:
I suggest using a dedicated "request" struct (`RpcValidatorsOrderedRequest`, see primitives/rpc.rs for inspiration), so if we happen to decide to add more options to the request, we can do it in a non-breaking way.

```suggestion
        let RpcValidatorsOrderedRequest { block_id } = parse_params(params)?;
```